README.md

```diff
@@ -21,11 +23,11 @@ Add the package to your `composer.json` and run `composer update`.
 
 Add the service provider in `config/app.php`:
 
-    'Anchu\Ftp\FtpServiceProvider',
+    Anchu\Ftp\FtpServiceProvider::class,
```

Comment:
I think this won't work for everyone.
The special ::class constant are available as of PHP 5.5.0 but laravel 5.0 supports PHP >= 5.4.